### PR TITLE
Display avatar/icon in single value and in options for multi and single

### PR DESCRIPTION
**Pull request by hadasfa**
צילום מסך 2022-11-21 ב-11 38 07
צילום מסך 2022-11-21 ב-11 38 17
in last picture the white ring is from the photo and not the css 
צילום מסך 2022-11-21 ב-11 38 26

### CODE REVIEW

File: src/components/Chips/Chips.tsx [hunk 1]
```diff
@@ -110,7 +110,7 @@ const Chips: VibeComponent<ChipsProps, HTMLElement> & {
           <Avatar
             withoutBorder
             className={cx(styles.avatar, styles.left)}
-            customSize={16}
+            customSize={18}
```
Comment (hadasfa) [inline src/components/Chips/Chips.tsx:113]: design review request

File: src/components/Dropdown/Dropdown.jsx [hunk 2]
```diff
@@ -253,10 +253,10 @@ const Dropdown = ({
         Input,
         Option,
         Control,
-        ...(finalValueRenderer && { SingleValue }),
+        SingleValue,
```
Comment (hadasfa) [inline src/components/Dropdown/Dropdown.jsx:256]: we now set a single value component also when we did not receive a custom renderer (for adding an avatar or icon)

File: src/components/Dropdown/components/singleValue/singleValue.jsx [hunk 3]
```diff
@@ -1,13 +1,20 @@
...
-  if (!Renderer) return null;
+  const { Renderer, data, children } = props;
+
+  const value = Renderer ? <Renderer {...data} /> : <ChildrenContent data={data}>{children}</ChildrenContent>;
   return (
-    <components.SingleValue {...props} className="dropdown-wrapper__single-value--reset">
-      <Renderer {...data} />
+    <components.SingleValue
+      {...props}
+      className={cx(styles.singleValue, "dd", "dropdown-wrapper__single-value--reset")}
```
Comment (SergeyRoyt) [inline src/components/Dropdown/components/singleValue/singleValue.jsx:15]: what's `dd`?

Comment (hadasfa) [reply src/components/Dropdown/components/singleValue/singleValue.jsx]: removed

**Review by SergeyRoyt — APPROVED** [review]
Great feature 👍

**Pull request merged**